src/components/20-molecules/accordion/index.js

```diff
@@ -8,12 +7,13 @@ import { classMap } from 'lit/directives/class-map.js';
 import { defineVersioned } from '../../../utils/component-versioning';
 import applyDefaults from '../../../utils/apply-defaults';
 import { sanitizeSVG } from '../../../utils/sanitize';
+import debounce from '../../../utils/debounce';
 
 import styles from './index.scss';
 
-class AXAAccordeon extends LitElement {
+class AXAAccordion extends LitElement {
   static get tagName() {
-    return 'axa-accordeon';
+    return 'axa-accordion';
```

Comment:
This requires a **big fat warning** when announcing the new version on Slack's patterns-lib-devs!!! What if somebody has baked the old name into his app?

Also, we need to release as semver major version.